### PR TITLE
Modified user/login endpoint, now it returns studentID on authenticated

### PR DESCRIPTION
**Pull request by KevenLYF**

### CODE REVIEW

File: backend/logic/users/users.js [hunk 1]
```diff
@@ -72,15 +72,14 @@ var insertAdminUser = async (username, password, email, id) => {
...
-      username
-    ]);
+    await connection.query(
+      "INSERT INTO users (username, email, password) VALUES(?, ?, ?);",
+      [username, email, hash]
+    );
+    await connection.query(
+      "INSERT INTO staff_members (staff_id, username) VALUES(?, ?);",
+      [id, username]
+    );
```
Comment (KevenLYF) [inline backend/logic/users/users.js]: Ignore this part, just auto-formatting. No changes involved here

**Comment by christiaua** [comment]
Maybe also return the name if it's not too hard to change, like that we don't need another endpoint to display the name in the header bar. Task: `Display the user's name in the Header Bar`

**Comment by Ty-Won** [comment]
@KevenLYF will you be adding the name as @christiaua suggested? If so I will wait on that before merging

**Comment by KevenLYF** [comment]
@christiaua  @Ty-Won  Sure that's a good idea, sry I've been too busy didn't see the reply. I'll try to finish that by tonight.

**Comment by KevenLYF** [comment]
@christiaua  now it returns JSON `{"student_id":279263971,"username":"studentUser1"}`

**Comment by christiaua** [comment]
After the meeting discussion, return the student's `email`

**Pull request merged**